### PR TITLE
add Dracula Theme

### PR DESCRIPTION
**Pull request by joesdu**
Add [Dracula Theme](https://github.com/dracula)

### CODE REVIEW

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs [hunk 1]
```diff
@@ -41,10 +41,15 @@ public class AnsiConsoleTheme : ConsoleTheme
         public static AnsiConsoleTheme Literate { get; } = AnsiConsoleThemes.Literate;
 
         /// <summary>
-        /// A theme in the style of the original <i>Serilog.Sinks.Literate</i> using only standard 16 terminal colors that will work on light backgrounds.
+        /// A theme in the style of the original <i>Serilog.Sinks.Sixteen</i> using only standard 16 terminal colors that will work on light backgrounds.
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs:44]: This change is incorrect, the project was called `Serilog.Sinks.Literate`

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs [hunk 2]
```diff
@@ -41,10 +41,15 @@ public class AnsiConsoleTheme : ConsoleTheme
...
         /// </summary>
         public static AnsiConsoleTheme Sixteen { get; } = AnsiConsoleThemes.Sixteen;
 
+        /// <summary>
+        /// A theme in the style of the original <i>Serilog.Sinks.Dracula</i>.
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs:49]: This seems like a placeholder comment that should be closer to the description added to the README

**Review by nblumhardt — COMMENTED** [review]
Thanks for submitting this. Could you please include some screenshots from a couple of terminals for illustration? Thanks!

**Comment by nblumhardt** [comment]
Hi Joe! I should clarify, sorry - I meant some screenshots in the PR description, not in the repository itself. My apologies for the confusion.

**Comment by joesdu** [comment]
> Hi Joe! I should clarify, sorry - I meant some screenshots in the PR description, not in the repository itself. My apologies for the confusion.

DraculaTheme

**Comment by joesdu** [comment]
It has been adjusted as much as possible to match the [color values of the Dracula Theme](https://draculatheme.com/contribute#color-palette).

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs [hunk 3]
```diff
@@ -45,6 +45,11 @@ public class AnsiConsoleTheme : ConsoleTheme
         /// </summary>
         public static AnsiConsoleTheme Sixteen { get; } = AnsiConsoleThemes.Sixteen;
 
+        /// <summary>
+        /// A dark theme based on the popular Dracula Theme
```
Comment (sungam3r) [inline src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/AnsiConsoleTheme.cs:49]:
```suggestion
        /// A dark theme based on the popular Dracula Theme.
```

**Comment by nblumhardt** [comment]
Hi @joesdu, thanks for all of the effort.

The theme still appears to have some contrast issues; I noticed that Dracula uses the white-ish shade for foreground text, and the unreadable gray is "comment"; I wonder if the output would be more readable if the messages were rendered in the white, and the cyan shade used for stack trace continuation lines?

(This might still leave green for INF?)

**Comment by joesdu** [comment]
> Hi @joesdu, thanks for all of the effort.
> 
> The theme still appears to have some contrast issues; I noticed that Dracula uses the white-ish shade for foreground text, and the unreadable gray is "comment"; I wonder if the output would be more readable if the messages were rendered in the white, and the cyan shade used for stack trace continuation lines?
> 
> (This might still leave green for INF?)

Yes, there are problems when it's not a dark theme.

**Comment by bartelink** [comment]
Sad to see this closed @joesdu; it's definitely not my department but these problems do seem surmountable?
Ultimately Nick is just asking these questions from the point of view of getting a truly workable implementation on the first try - getting obvious tweaking done in one shot rather than as a series of chaser releases...

**Comment by joesdu** [comment]
> Sad to see this closed @joesdu; it's definitely not my department but these problems do seem surmountable? Ultimately Nick is just asking these questions from the point of view of getting a truly workable implementation on the first try - getting obvious tweaking done in one shot rather than as a series of chaser releases...

Dracula Theme does not seem to have an example of a light theme. I checked the theme configurations of Windows Terminal and Notepad++, and it only contains dark themes, so it may not be applicable to the light theme version. So I closed this PR, if it is used If you have a dark theme and need to customize the theme color, you can use the following code to achieve it.
```csharp
builder.Host.UseSerilog((hbc, lc) =>
{
    const LogEventLevel logLevel = LogEventLevel.Information;
    lc.ReadFrom.Configuration(hbc.Configuration)
      .MinimumLevel.Override("Microsoft", logLevel)
      .MinimumLevel.Override("System", logLevel)
      .Enrich.FromLogContext()
      .WriteTo.Async(wt =>
      {
          if (hbc.HostingEnvironment.IsDevelopment())
          {
              //wt.SpectreConsole();
              wt.Debug();
          }
          wt.Console(theme: new AnsiConsoleTheme(new Dictionary<ConsoleThemeStyle, string>
          {
              [ConsoleThemeStyle.Text] = "\x1b[38;5;151m",
              [ConsoleThemeStyle.SecondaryText] = "\x1b[38;5;245m",
              [ConsoleThemeStyle.TertiaryText] = "\x1b[38;5;244m",
              [ConsoleThemeStyle.Invalid] = "\x1b[38;5;214m",
              [ConsoleThemeStyle.Null] = "\x1b[38;5;248m",
              [ConsoleThemeStyle.Name] = "\x1b[38;5;141m",
              [ConsoleThemeStyle.String] = "\x1b[38;5;168m",
              [ConsoleThemeStyle.Number] = "\x1b[38;5;141m",
              [ConsoleThemeStyle.Boolean] = "\x1b[38;5;248m",
              [ConsoleThemeStyle.Scalar] = "\x1b[38;5;119m",
              [ConsoleThemeStyle.LevelVerbose] = "\x1b[37m",
              [ConsoleThemeStyle.LevelDebug] = "\x1b[37m",
              [ConsoleThemeStyle.LevelInformation] = "\x1b[37;1m",
              [ConsoleThemeStyle.LevelWarning] = "\x1b[38;5;208m",
              [ConsoleThemeStyle.LevelError] = "\x1b[38;5;197m\x1b[48;5;238m",
              [ConsoleThemeStyle.LevelFatal] = "\x1b[38;5;197m\x1b[48;5;238m"
          }));
      });
});
```

**Pull request closed without merging**